stdlib/2/xmlrpclib.pyi

```diff
@@ -0,0 +1,193 @@
+
+_Unmarshaller = TypeVar('_Unmarshaller')
+_timeTuple = Tuple[int, int, int, int, int, int, int, int, int]
+# Represents types that can be compared against a DateTime object
+_dateTimeComp = Union[AnyStr, DateTime, datetime, _timeTuple]
+# A "host description" used by Transport factories
+_hostDesc = Union[str, Tuple[str, Mapping[Any, Any]]]
+
+def escape(s: AnyStr, replace: Callable[[AnyStr, AnyStr, AnyStr], AnyStr] = ...) -> AnyStr: ...
+
+MAXINT = ...  # type: int
+MININT = ...  # type: int
+PARSE_ERROR = ...  # type: int
+SERVER_ERROR = ...  # type: int
+APPLICATION_ERROR = ...  # type: int
+SYSTEM_ERROR = ...  # type: int
+TRANSPORT_ERROR = ...  # type: int
+NOT_WELLFORMED_ERROR = ...  # type: int
+UNSUPPORTED_ENCODING = ...  # type: int
+INVALID_ENCODING_CHAR = ...  # type: int
+INVALID_XMLRPC = ...  # type: int
+METHOD_NOT_FOUND = ...  # type: int
+INVALID_METHOD_PARAMS = ...  # type: int
+INTERNAL_ERROR = ...  # type: int
+
+class Error(Exception): ...
+
+class ProtocolError(Error):
+    url = ...  # type: str
+    errcode = ...  # type: int
+    errmsg = ...  # type: str
+    headers = ...  # type: Mapping[str, str]
+    def __init__(self, url, errcode, errmsg, headers) -> None: ...
+
+class ResponseError(Error): ...
+
+class Fault(Error):
+    faultCode = ...  # type: Any
+    faultString = ...  # type: str
+    def __init__(self, faultCode: Any, faultString: str, **extra: Any) -> None: ...
+
+boolean = ...  # type: Type[bool]
+Boolean = ...  # type: Type[bool]
+
+class DateTime:
+    value = ...  # type: str
+    def __init__(self, value: Union[AnyStr, datetime, float, int, _timeTuple] = ...) -> None: ...
+    def make_comparable(self, other: _dateTimeComp) -> Tuple[_dateTimeComp, _dateTimeComp]: ...
+    def __lt__(self, other: _dateTimeComp) -> bool: ...
+    def __le__(self, other: _dateTimeComp) -> bool: ...
+    def __gt__(self, other: _dateTimeComp) -> bool: ...
+    def __ge__(self, other: _dateTimeComp) -> bool: ...
+    def __eq__(self, other: _dateTimeComp) -> bool: ...
+    def __ne__(self, other: _dateTimeComp) -> bool: ...
+    def timetuple(self) -> struct_time: ...
+    def __cmp__(self, other: _dateTimeComp) -> int: ...
+    def decode(self, data: Any) -> None: ...
+    def encode(self, out: IO) -> None: ...
+
+class Binary:
+    data = ...  # type: str
+    def __init__(self, data: Optional[str] = ...) -> None: ...
+    def __cmp__(self, other: Any) -> int: ...
+    def decode(self, data: str) -> None: ...
+    def encode(self, out: IO) -> None: ...
+
+WRAPPERS = ...  # type: tuple
+
+# Still part of the public API, but see http://bugs.python.org/issue1773632
+FastParser = ...  # type: None
+FastUnmarshaller = ...  # type: None
+FastMarshaller = ...  # type: None
+
+# I'm going to assume, quite reasonably, that we'll have xml.parsers.expat
+# available.
+class ExpatParser:
+    def __init__(self, target: _Unmarshaller) -> None: ...
+    def feed(self, data: str): ...
+    def close(self): ...
+
+class SlowParser:
```

Comment:
So I agree that we should make SlowParser inherit from xmllib.XMLParser, but that makes the file fail to import in mypy.
I think it might be better to leave it without the base class for now. It can be added later if stubs for xmllib are added.